CollAction/Views/Home/FAQ.cshtml

```diff
@@ -7,9 +7,8 @@
 }
 
 @section styles {
-    <link rel="stylesheet" href="~/css/faq.css" />
+        <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.4.1/css/all.css">
```

Comment:
Formatting. Also, could you keep the integrity attribute? As discussed in the call, that does help.